runtime/domain.c

```diff
@@ -1533,7 +1521,6 @@ void caml_handle_gc_interrupt(void)
     CAML_EV_END(EV_INTERRUPT_REMOTE);
   }
 
-  caml_reset_young_limit(Caml_state);
```

Comment:
Would you be able to explain why this is necessary in `caml_handle_gc_interrupt`?

Reply:
Ok, that makes sense. Thanks.